ansible/README.md

```diff
@@ -100,8 +100,19 @@ db_password=
 db_protocol=
 db_host=
 db_port=
+
+[controller]
+db_username=
+db_password=
+
+[invoker]
+db_username=
+db_password=
 ```
 
+The section [db_creds] is to configure the admin access. The sections controller and invoker are for the authentication that these components are using.
+You can use any username and password you like here. They will be generated automatically later, using the admin credentials from above.
+
```

Comment:
when using cloudant, do need to create an empty `db_local.ini` file as well as run `ansible-playbook -i environments/<environment> couchdb.yml --tags ini` ?

Reply:
Using the ini tag is described a few lines below.

If you can use a command or have to create a file is written in the line below.

Or what exactly do you mean?

Reply:
i think i understand my confusion.  some of the existing text is a bit unclear as to whether or not the user is expected to manual create/edit the `db_local.ini` file or just run the playbook.

the sentence `You can use any username and password you like here` implies manual editing.  later the `couchdb.yml` playbook is mentioned as a way to generate the `db_local.ini` file; but since the list of environment variables does not include values corresponding to the controller and invoker .ini section, it's not clear how/if the playbook will generate these sections.  i did run the playbook, and it does indeed automatically generate the `[controller]` and `[invoker]` sections with default `db_username` and `db_password` values, but the doc does not mention this.


the sentence `You can use any username and password you like here.` seems to imply that the db_local.ini file is manually created/updated via an editor.

Reply:
`They will be generated automatically later`.. it might be clearer to mention where (i.e in the db) these will be generated, and what the "later" phase is (i.e. during db initialization)